### DISCUSSION TITLE
Problem with PLOT file when creating in Hybrid Mode

### DISCUSSION THREAD

**Discussion by junkersozaur** [Chia-Network/chia-blockchain#18994] [Support]
Hello guys,
Sorry for maybe lame question but I'm new for this.
Have a good hardware ( 160GB RAM) and want to create PLOT with Hybrid Mode (still have 140GB RAM free) on BladeBit 3.1.0 (CUDA) the PLOT file have around ~91GB and not showing up after complete.
Even when adding manually.
Normal PLOT files that works have ~101,4GB showing and working without a problem.
Is there something about Hybrid Mode?
Thanks in advance.

**Comment by junkersozaur** [comment]
![obraz (1)](https://github.com/user-attachments/assets/8165d0b9-0658-4285-afb6-0cdaf5cdb3bf)

**Comment by BrandtH22** [comment]
Hey @junkersozaur , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

The screenshot shows that the plot is still being created. Did it fully complete?

**Reply by junkersozaur** [reply]
Yeah, it is completed but not visible.
Working files as mentioned have more than 100GB - these created by CUDA have ~91GB after complete and are not visible.

**Reply by BrandtH22** [reply]
- What OS are you running?
- What compression level are you trying to create?
- What GPU are you using?

**Reply by junkersozaur** [reply]
Hi,
Thanks for replay.
OS Windows 11 24H2
Without compression.
GPU is RTX3090.

**Reply by BrandtH22** [reply]
Hey @junkersozaur , if you upgraded to win 24H2 after installing chia please perform a clean install of chia then try the plotting process again. 

If not can you provide your log file from `~/.chia/mainnet/log/debug.log`?

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
